geth/jail/jail_test.go

```diff
@@ -149,13 +159,7 @@ func (s *JailTestSuite) TestJailRPCAsyncSend() {
 			wg.Add(1)
 			defer wg.Done()
 
-			_, err = cell.Run(`
-				_status_catalog.commands.sendAsync({
-					"from": "` + TestConfig.Account1.Address + `",
-					"to": "0xf82da7547534045b4e00442bc89e16186cf8c272",
-					"value": "0.000001"
-				})
-			`)
```

Comment:
Less Readable in the sense of moving the script source to a variable or as to leaving it here? @tiabc